ganga/GangaLHCb/Lib/LHCbDataset/LHCbDataset.py

```diff
@@ -55,7 +57,7 @@ class LHCbDataset(GangaDataset):
                       'getLFNs', 'getFileNames', 'getFullFileNames',
                       'difference', 'isSubset', 'isSuperset', 'intersection',
                       'symmetricDifference', 'union', 'bkMetadata',
-                      'isEmpty', 'hasPFNs', 'getPFNs']  # ,'pop']
+                      'isEmpty', 'hasPFNs', 'getPFNs', 'setReference', 'createIndex']  # ,'pop']
```

Comment:
Should these new methods be exposed to the end user?

Reply:
Ah no, that was for my testing. I'll remove that.